massgov/mayflower
# DP-19330 Expand the menu button width with "close" label

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/develop/docs/for-developers/changelog-instructions.md)


## Description

Expand the menu button width to fill the available space on the blue bar when it's labeled as "close" with the screen size 840px and smaller.

## Related Issue / Ticket

- [JIRA issue](https://jira.mass.gov/browse/DP-19330)

## Steps to Test


1. Go to http://localhost:3000/patterns/05-pages-Homepage-hamburger/05-pages-Homepage-hamburger.html.
2. Check the menu button widths in different screen sizes.
- With the screen width bigger than 840 px, the button width stays same.
- With the screen width between 621 px and 840 px, the button width with the label "close" expands to any available space on the blue bar. As the menu close, the width goes back to its initial size.
- With the screen width 620 px and smaller, the button width with the label "close" expands to any available space on the blue bar, and it doesn't interfere with the search button. As the menu close, the width goes back to its initial size.

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


*

#### @TODO


*

#### Today I learned...
